app/javascript/src/components/Dashboard/Contacts/utils.jsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+
+import { MenuHorizontal } from "neetoicons";
+import { Dropdown, Avatar, Typography } from "neetoui";
+
+const renderNameRoleAvatar = (name, { role }) => (
+  <div className="flex space-x-4">
+    <Avatar
+      user={{
+        imageUrl: "https://xsgames.co/randomusers/avatar.php?g=pixel",
+      }}
+    />
+    <div>
+      <Typography style="h5">{name}</Typography>
+      <Typography className="neeto-ui-text-gray-600" style="body3">
+        {role}
+      </Typography>
+    </div>
+  </div>
+);
+
+const renderDropdown = () => (
```

Comment:
It's not simply a dropdown. It's a dropdown rendering certain actions. Let's name it appropriately.

```suggestion
const renderActionDropdown = () => (
```